### PR TITLE
Updating trackAnalyticsEvent() calls in numerous components (Part 1)

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request updates the information passed to `trackAnalyticsEvent()` in numerous components to follow the updates established in #1897. 

### How should this be reviewed?

👀 

### Relevant tickets

References [Pivotal #170710129](https://www.pivotaltracker.com/story/show/170710129).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: resources/assets/components/pages/AccountPage/Profile.js [hunk 1]
```diff
@@ -60,21 +60,15 @@ const Profile = props => (
           <a
             href="/deauthorize"
             className="text-gray-600 font-normal underline"
-            // TODO: Fix this analytics item
```
Comment (mendelB) [inline resources/assets/components/pages/AccountPage/Profile.js]: Did we mean to update this since this isn't really a navigation link anymore?

Comment (weerd) [reply resources/assets/components/pages/AccountPage/Profile.js]: Ah, right. Maybe we should switch it to being a `category` of "authentication"? cc @DFurnes thoughts as well?

Comment (weerd) [reply resources/assets/components/pages/AccountPage/Profile.js]: Might not change it just yet, cause I think the Data Team will need to know but I can add a comment suggesting to switch the category to "authentication" 🤔

Comment (DFurnes) [reply resources/assets/components/pages/AccountPage/Profile.js]: Yeah, I agree it makes sense to rename (as long as everyone's on the same page)! 👍

**Pull request merged**